cmd/textile/main.go

```diff
@@ -47,9 +51,9 @@ var (
 			Key:      "scope",
 			DefValue: "",
 		},
-		"addrApi": {
-			Key:      "addr.api",
-			DefValue: "/ip4/127.0.0.1/tcp/3006",
+		"apiTarget": {
```

Comment:
gRPC calls the connection string a "target", inferring ports from protocols (http2). Seems most accurate to bubble the language to here.